content/concepts/introduction/protocols/ping.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Ping"
+description: "The ping protocol is a simple request response protocol."
+weight: 20
+---
+
+## What is Ping?
+
+Ping is a network utility used to test a node's reachability.
+The ping protocol is a simple request-response protocol that measures the RTT
```

Comment:
It’s not request-response, the stream is reused for future pings.

Reply:
The [spec](https://github.com/libp2p/specs/blob/master/ping/ping.md#protocol) mentions that is is request-response. I believe it is better to introduce the protocol as a liveness check, or state non-ICMP.